.travis.yml

```diff
@@ -11,9 +11,9 @@ cache:
 before_script:
   - cp .env.travis .env
   - mysql -e 'create database `header-checker-rest`;'
-  - composer self-update
   - composer install --no-interaction
 
 script:
   - composer validate --strict
-  - vendor/bin/phpunit --coverage-text
+  - vendor/bin/security-checker security:check --end-point=http://security.sensiolabs.org/check_lock
```

Comment:
HTTP... 😱  But we discussed it, and unfortunately (and shamefully!) it's Sensio Labs fault.

Can't we add a comment in `.travis.yml`?